examples/langchain/ask_similar_query.py

```diff
@@ -28,10 +28,19 @@ def format_docs(docs):
 load_dotenv()
 
 prompt = hub.pull("rlm/rag-prompt")
-llm = ChatOpenAI(model_name="gpt-3.5-turbo", temperature=0)
+llm = AzureChatOpenAI(
+    api_version="2023-12-01-preview",
+    azure_endpoint="https://gpt-testing-custom-data1.openai.azure.com",
```

Comment:
I think we want these in the `.env` file. I don't think they're secretive, but they still seem like things we don't want to put in a public place.